### PR TITLE
salt pr: robots no longer have balls

### PR DESCRIPTION
**Pull request by lolman360**
## About The Pull Request

robots no longer can be negatively affected by being punched in the balls

i should probably make them resist most things in unarmed melee combat but they're already stronger as it is

## Why It's Good For The Game

1. salt pr
2. protean buff
3. immersion

## Changelog
:cl:
balance: you can no longer kick robots in the nuts
/:cl:

### CODE REVIEW

**Comment by Captain277** [comment]
Getting some compile errors via Travis.

**Comment by Ghostcoffeee** [comment]
There's an extra closing bracket, so that's what's probably causing it

File: code/modules/mob/living/carbon/human/unarmed_attack.dm [hunk 1]
```diff
@@ -74,8 +74,8 @@ var/global/list/sparring_attack_cache = list()
 					if(prob(50))
 						target.setDir(reverse_dir[target.dir])
 					target.apply_effect(attack_damage * 0.4, WEAKEN, armour)
-			if(BP_GROIN)
-				target.visible_message("<span class='warning'>[target] looks like [TT.he] [TT.is] in pain!</span>", "<span class='warning'>[(target.gender=="female") ? "Oh god that hurt!" : "Oh no, not your[pick("testicles", "crown jewels", "clockweights", "family jewels", "marbles", "bean bags", "teabags", "sweetmeats", "goolies")]!"]</span>") // I see no easy way to fix this for non-organic or neuter characters.
+			if(BP_GROIN && !isSynthetic(target)))
```
Comment (buffyuwu) [inline code/modules/mob/living/carbon/human/unarmed_attack.dm:77]: remove a )

**Comment by lolman360** [comment]
@Captain277 who the fuck testmerged a pr that failed unit tests

**Pull request merged**